### PR TITLE
require -> use

### PR DESCRIPTION
**Pull request by AndreaCrotti**
Just replacing a few `use` to use `require` instead

### CODE REVIEW

**Comment by lvh** [comment]
Looks like this broke something: https://travis-ci.org/cloverage/cloverage/jobs/493176952#L515

File: cloverage/src/cloverage/instrument.clj [hunk 1]
```diff
@@ -1,15 +1,9 @@
...
-        [clojure.string  :only [split]])
-  (:require [clojure.set :as set]
-            [clojure.test :as test]
-            [clojure.tools.logging :as log]
-            [cloverage.rewrite :refer [unchunk]]
+  (:require [clojure.tools.logging :as log]
             [clojure.tools.reader :as r]
+            [cloverage.rewrite :refer [unchunk]]
             [riddley.walk :refer [macroexpand-all]]
-            [cloverage.debug :refer :all]
```
Comment (lvh) [inline cloverage/src/cloverage/instrument.clj]: This line appears to be deleted but `clojure,debug` does have some symbols we use here, at least tprnl. (I'd be fine with a `:refer` or an `:as`.)

**Comment by lvh** [comment]
I added a comment inline about where the breakage is. Thanks for working on this!

File: cloverage/src/cloverage/instrument.clj (hunk 1 above)
Comment (AndreaCrotti) [reply cloverage/src/cloverage/instrument.clj]: Ok that's fixed

**Comment by AndreaCrotti** [comment]
It still fails now but only on the JDK9, which is some progress I guess 😄

**Comment by lvh** [comment]
Yeah that's an unrelated issue. Thanks!

**Pull request merged**